lib/routes/data/styles.js

```diff
@@ -63,4 +63,21 @@ module.exports = {
       ],
     ],
   },
+  breadcrumb: {
```

Comment:
This can just be an array called `breadcrumbs` (no need for `item` sub-array), then below you could just use `{% for item in breadcrumbs %}`

This might cause an issue with your macro's name being `breadcrumbs`, but it might not...